synapse/storage/appservice.py

```diff
@@ -433,3 +437,7 @@ def get_new_events_for_appservice_txn(txn):
         events = yield self._get_events(event_ids)
 
         defer.returnValue((upper_bound, events))
+
+
+class ApplicationServiceTransactionStore(ApplicationServiceTransactionWorkerStore):
+    pass
```

Comment:
Should I remove this and replace it everywhere with `ApplicationServiceTransactionWorkerStore`?

Reply:
a comment to clarify mightn't hurt.